### DISCUSSION TITLE
Power Utilization implementation

### DISCUSSION THREAD

**Discussion by David78330** [netbox-community/netbox#6041] [Q&A]
Hi Netbox users,

I have an Netbox version 2.9.10 running with this kind of power implementation tracked from this schema.
https://netbox.readthedocs.io/en/stable/core-functionality/power/

```
=================================================================================
Power_Panel_A |--> Power_Feed_A1 --> Device_PDU_1 --> PDU 1.1 |--> Server 1 PSU 1
              |                            |      --> PDU 1.2 |--> Server 2 PSU 1
              |                            |--> etc...
              |         
Power_Panel_B |--> Power_Feed_B1 --> Device_PDU_2 --> PDU 2.1 |--> Server 1 PSU 2
              |                            |      --> PDU 2.2 |--> Server 2 PSU 2
              |                            |--> etc...
==================================================================================		
```  

I've set :
- Characteristics on Power_Feed_1 and Power_Feed_2 (AC/240V/20A). It calculates 3840VA of possible power utlization.
- On Device_PDU_1, one "Power Ports" connected to Power_Feed_1 for the input and multiple "Power Outlets" where I will connect the Server X PSU 1.
- On Device_PDU_2, one "Power Ports" connected to Power_Feed_2 for the input and multiple "Power Outlets" where I will connect the Server X PSU 2.
- On each "Server" devices have 2 "Power Ports" with Draw of 500W and connected to "Power Outlets" of the Device_PDU_X.

My issue with this configuration is all my "Power Utilization" bar remains at 0%.
Did I do something wrong?

I've looked to some reported issues but not found solution (https://github.com/netbox-community/netbox/issues/3377 or https://github.com/netbox-community/netbox/issues/5559).

Thanks for your help.

Regards
David

**Comment by David78330** [comment]
Hello,

Nobody can help me with this issue?

Regards,
David

**Comment by bitcollector1** [comment]
What does your cable trace show?  I'll post screenshots of my working model. Hope it helps. 

![Panel-to-feed](https://user-images.githubusercontent.com/50723251/115812955-3345a400-a3a7-11eb-8508-990bef8acd8c.png)

![PDU to Server](https://user-images.githubusercontent.com/50723251/115812971-39d41b80-a3a7-11eb-87d8-7f345420595e.png)

![rack-power](https://user-images.githubusercontent.com/50723251/115813024-49536480-a3a7-11eb-9188-adecf610ab34.png)
